### DISCUSSION TITLE
Flycast-VL: often recommended, what is it?

### DISCUSSION THREAD

**Discussion by XFer012** [flyinghead/flycast#1592] [Q&A] [answered]
Hello,
got a few ARM64 retro consoles (Odroid N2, Anbernic RG35XX-H, Trimui Smart Pro) and have some performance issues with Dreamcast games.
More often than not, I am adviced to switch Retroarch emulation core from "flycast" to "flycast vl" or "flycast xtreme".
I have no such options in my Retroarch core menu, so I'm wondering if they are optimized branches of Flycast?
Where may I find these cores?

Thanks!

**Comment by flyinghead** [comment, marked as answer]
I don't know what these builds are but I doubt they are useful. Flycast is already highly optimized, and there's no easy optimization that hasn't been done. Plus these builds are very likely outdated. I found an old github repo with a flycast xtreme core, last update 5 years ago (!), no source code included (in violation of the GPL licence)...

**Reply by Airwalk16** [reply]
Hey flyinghead. I dint really know how else to get a hold of you. But my flycast isn't able to oull boxart for a few games. Is there a list of games with the Number associated with each game so we k ow what we need to name them so they will work if we put a downloaded image into the data>boxart folder?

I need:

Chaos field
Animal Basket
Brick People
BLOKPONG
GiantGram2
GiantGram2000
Illvello
Mamonoro
Mushiking2k5
Rhythm Tengoku
Shooting Love 2007
Touch De Uno 2

Thanks if you can!

You can even email me at airwalk1616@gmail.com

**Reply by Airwalk16** [reply]
I went ahead and added all of those missing games and boxarts to thegamesdb.net but they aren't showing up on my flycast still. I guess because I didn't add any number to any field? I saw there was a UID field but I had no idea what that was. I'm curious if that's why?

**Comment by XFer012** [comment]
Thank you!
